### ISSUE TITLE
Form submit redirect to another page (for google tracking) worked on previous version

### ISSUE THREAD

**Issue by oonielin** [a3rev/woocommerce-email-inquiry-cart-options-lite#5]
Hi, I'm a newbie that needs help. We currently have a script that would redirect to a thank you page once the inquiry form has been sent.
The code was added by someone else and I am now trying to get it to work but don't know javascript nor php....why? because we can't get hold of the developer who helped us and have been let down by a few developers that I hired - yay!
Here is the older version of the plugin working with the redirect:
https://myspace.co.za/product-category/metal-beds/ - click product enquiry and submit the form.  Then it redirects to  https://myspace.co.za/thank-you/
This is the script in class-wc-email-inquiry-hook.php : 
Any help would be much appreciated please? Please in your reply pretend i'm 5years old and spell it out for me. Thanks Lindsay

<script type="text/javascript">
(function($){
	$(function(){
		var ajax_url = "<?php echo admin_url( 'admin-ajax.php', 'relative' ); ?>";
		$(document).on("click", ".wc_email_inquiry_popup_button", function(){
			var product_id = $(this).attr("product_id");
			var product_name = $(this).attr("product_name");
		<?php
			$wc_email_inquiry_popup_type = $wc_email_inquiry_global_settings['inquiry_popup_type'];
			if ($wc_email_inquiry_popup_type == 'colorbox') {
		?>
			var popup_wide = 520;
			var center_on_scroll = true;
			if ( ei_getWidth()  <= 568 ) { 
				popup_wide = '100%';
				center_on_scroll = false;
			}
			$.colorbox({
				href		: ajax_url+"?action=wc_email_inquiry_popup&product_id="+product_id+"&security=<?php echo $wc_email_inquiry_popup; ?>",
				className	: 'email_inquiry_cb',
				opacity		: 0.85,
				scrolling	: true,
				initialWidth: 100,
				initialHeight: 100,
				innerWidth	: popup_wide,
				//innerHeight	: 500,
				maxWidth  	: '100%',
				maxHeight  	: '90%',
				returnFocus : true,
				transition  : 'none',
				speed		: 300,
				fixed		: center_on_scroll
			});
		<?php } else { ?> 
			var popup_wide = 520;
			var center_on_scroll = true;
			if ( ei_getWidth()  <= 568 ) { 
				popup_wide = '95%';
				center_on_scroll = false;
			}
			$.fancybox({
				href: ajax_url+"?action=wc_email_inquiry_popup&product_id="+product_id+"&security=<?php echo $wc_email_inquiry_popup; ?>",
				centerOnScroll : center_on_scroll,
				transitionIn : 'none', 
				transitionOut: 'none',
				easingIn: 'swing',
				easingOut: 'swing',
				speedIn : 300,
				speedOut : 0,
				width: popup_wide,
				autoScale: true,
				autoDimensions: true,
				height: 460,
				margin: 0,
				maxWidth: "95%",
				maxHeight: "80%",
				padding: 10,
				overlayColor: '#666666',
				showCloseButton : true,
				openEffect	: "none",
				closeEffect	: "none"
			});
		<?php } ?>
		});
		
		$(document).on("click", ".wc_email_inquiry_form_button", function(){
			if ( $(this).hasClass('wc_email_inquiry_sending') ) {
				return false;
			}
			$(this).addClass('wc_email_inquiry_sending');
			
			var product_id = $(this).attr("product_id");
			var your_name = $("#your_name_"+product_id).val();
			var your_email = $("#your_email_"+product_id).val();
			var your_phone = $("#your_phone_"+product_id).val();
			var your_location = $("#your_location_"+product_id).val();
			var your_message = $("#your_message_"+product_id).val();
			var send_copy = 0;
			
			var wc_email_inquiry_error = "";
			var wc_email_inquiry_have_error = false;
			var filter = /^([a-zA-Z0-9_\.\-\+])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
			
			if (your_name.replace(/^\s+|\s+$/g, '') == "") {
				wc_email_inquiry_error += "<?php _e('Please enter your Name', 'woocommerce-email-inquiry-cart-options' ); ?>\n";
				wc_email_inquiry_have_error = true;
			}
			if (your_email == "" || !filter.test(your_email)) {
				wc_email_inquiry_error += "<?php _e('Please enter valid Email address', 'woocommerce-email-inquiry-cart-options' ); ?>\n";
				wc_email_inquiry_have_error = true;
			}
			if (your_phone.replace(/^\s+|\s+$/g, '') == "") {
				wc_email_inquiry_error += "<?php _e('Please enter your Phone', 'woocommerce-email-inquiry-cart-options' ); ?>\n";
				wc_email_inquiry_have_error = true;
			}
			if (your_location.replace(/^\s+|\s+$/g, '') == "") {
				wc_email_inquiry_error += "<?php _e('Please enter your Location', 'woocommerce-email-inquiry-cart-options' ); ?>\n";
				wc_email_inquiry_have_error = true;
			}
			if (wc_email_inquiry_have_error) {
				$(this).removeClass('wc_email_inquiry_sending');
				alert(wc_email_inquiry_error);
				return false;
			}
			$("#wc_email_inquiry_loading_"+product_id).show();
			
			var data = {
				action: 		"wc_email_inquiry_action",
				product_id: 	product_id,
				your_name: 		your_name,
				your_email: 	your_email,
				your_phone: 	your_phone,
				your_location: 	your_location,
				your_message: 	your_message,
				send_copy:		send_copy,
				security: 		"<?php echo $wc_email_inquiry_action; ?>"
			};
			$.post( ajax_url, data, function(response) {
	wc_email_inquiry_response = $.parseJSON( response );
	$("#wc_email_inquiry_loading_"+product_id).hide();
	$("#wc_email_inquiry_content_"+product_id).html(wc_email_inquiry_response);
	<?php if ( $wc_email_inquiry_popup_type == 'colorbox' ) { ?>
	var height_cb = false;
	if ( ei_getWidth()  <= 568 ) { 
		height_cb = '90%';
	}
	$.colorbox.resize({
		height:		height_cb
	});
	<?php } ?>
}).done(function() {
	window.location.href = "https://www.myspace.co.za/thank-you/";
})
.fail(function() {
	window.location.href = "https://www.google.com/";
});
		});
	});
})(jQuery);
</script>
